packages/web/src/scss/components/Tooltip/README.md

```diff
@@ -224,14 +224,49 @@ on the `.Tooltip__arrow` to control it and prevent conflicts with the default CS
 </div>
 ```
 
-### Example
+### Floating UI functionality
 
-💻 Check our [example] that uses external library
-[Floating UI][floating-ui] (see the [JS source](./floating-ui-example.mjs)).
+To enable the Floating UI functionality, you need to activate the JS plugin, wrap your tooltip with an element having the `data-spirit-element="tooltip-wrapper"` data attribute, and add the `data-spirit-placement-controlled` attribute to your tooltip element.
```

Comment:
Wasn't it four steps? If yes, I'd put the instructions as a numbered list (three steps are OK in a sentence).

Reply:
Added 4th step - info about feature flag